src/index.html

```diff
@@ -5,8 +5,401 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com" >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="true"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+          src="images/logo.png"
+          alt="moyo logo"
+        >
+      </a>
+      <nav class="navigation">
+        <ul class="navigation__nav-list">
+          <li>
+            <a class="navigation__nav-link is-active">apple</a>
+          </li>
+          <li>
+            <a class="navigation__nav-link">samsung</a>
+          </li>
+          <li>
+            <a class="navigation__nav-link">smartphones</a>
+          </li>
+          <li>
+            <a class="navigation__nav-link" data-qa="nav-hover">laptops & computers</a>
+          </li>
+          <li>
+            <a class="navigation__nav-link">gadgets</a>
+          </li>
+          <li>
+            <a class="navigation__nav-link">tablets</a>
+          </li>
+          <li>
+            <a class="navigation__nav-link">photo</a>
+          </li>
+          <li>
+            <a class="navigation__nav-link navigation__nav-link--last">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <section class="catalog">
+        <div class="card catalog__card" data-qa="card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          >
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__product-code">Product code: 195434</p>
+
+          <div class="card__rate">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__reviews">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-title">Price:</p>
+
+            <p class="card__price-sum">$2,199</p>
+          </div>
+
+          <a
+            href="/basket"
+            class="card__link"
+            data-qa="card-hover"
+            >
+            buy
```

Comment:
```suggestion
            data-qa="card-hover"
          >
            buy
```

Reply:
now i have weird-tiny shift in one test, i cant find issue. 
and still a lot of questions in this task)